caddyhttp/proxy/proxy_test.go

```diff
@@ -710,13 +710,18 @@ func basicAuthTestcase(t *testing.T, upstreamUser, clientUser *url.Userinfo) {
 }
 
 func newFakeUpstream(name string, insecure bool) *fakeUpstream {
-	uri, _ := url.Parse(name)
+	if strings.Contains(name,"{") && strings.Contains(name,"{") {
```

Comment:
I think the second `strings.Contains` call was meant to be with a `}` instead of `{`.